### ISSUE TITLE
Dedicated server crashes immediately after FOB spawns initially

### ISSUE THREAD

**Issue by Goddler** [DUWS-R-Team/DUWS-R#158]
20:59:48 Game started.
Attempt to override final function - bis_fnc_storeparamsvalues_data
21:00:04 Warning: Adding units to a remote group is not safe. Please use setGroupOwner to change the group owner first.
21:00:13 Server: Object 3:18 not found (message Type_128)
21:00:13 Server: Object 3:19 not found (message Type_114)
21:00:13 Server: Object 3:25 not found (message Type_115)
21:00:13 Server: Object 3:22 not found (message Type_93)
21:00:13 Server: Object 3:23 not found (message Type_121)
21:00:13 Server: Object 3:28 not found (message Type_450)
21:00:13 Server: Object 3:24 not found (message Type_127)
21:00:13 Server: Object 3:31 not found (message Type_114)
21:00:13 Server: Object 3:29 not found (message Type_121)
21:00:13 Server: Object 3:30 not found (message Type_127)
21:00:13 Server: Object 3:40 not found (message Type_128)
21:00:13 Server: Object 3:34 not found (message Type_121)
21:00:13 Server: Object 3:35 not found (message Type_121)
21:00:13 Server: Object 3:36 not found (message Type_127)
21:00:13 Server: Object 3:37 not found (message Type_114)
21:00:13 Server: Object 3:38 not found (message Type_121)
21:00:13 Server: Object 3:39 not found (message Type_121)
21:00:13 Server: Object 3:46 not found (message Type_114)
21:00:13 Server: Object 3:41 not found (message Type_114)
21:00:13 Server: Object 3:42 not found (message Type_121)
21:00:13 Server: Object 3:43 not found (message Type_121)
21:00:13 Server: Object 3:44 not found (message Type_127)
21:00:13 Server: Object 3:45 not found (message Type_121)
21:00:13 Server: Object 3:61 not found (message Type_115)
21:00:13 Server: Object 3:55 not found (message Type_128)
21:00:13 Server: Object 3:56 not found (message Type_114)
21:00:13 Server: Object 3:57 not found (message Type_121)
21:00:13 Server: Object 3:58 not found (message Type_121)
21:00:13 Server: Object 3:59 not found (message Type_127)
21:00:13 Server: Object 3:60 not found (message Type_121)
21:00:13 Server: Object 3:51 not found (message Type_433)
21:00:13 Server: Object 3:49 not found (message Type_121)
21:00:14 Server: Object 3:50 not found (message Type_121)
21:00:14 Server: Object 3:53 not found (message Type_121)
21:00:14 Server: Object 3:52 not found (message Type_114)
21:00:14 Server: Object 3:54 not found (message Type_450)
21:00:14 Server: Object 3:70 not found (message Type_131)
21:00:14 Server: Object 3:64 not found (message Type_131)
21:00:14 Server: Object 3:65 not found (message Type_132)
21:00:14 Server: Object 3:66 not found (message Type_121)
21:00:14 Server: Object 3:67 not found (message Type_121)
21:00:14 Server: Object 3:68 not found (message Type_127)
21:00:14 Server: Object 3:69 not found (message Type_114)
21:00:14 Server: Object 3:71 not found (message Type_132)
21:00:14 Server: Object 3:72 not found (message Type_121)
21:00:14 Server: Object 3:73 not found (message Type_121)
21:00:14 Server: Object 3:74 not found (message Type_127)
21:00:14 Server: Object 3:75 not found (message Type_114)
21:00:14 Server: Object 3:81 not found (message Type_121)
21:00:14 Server: Object 3:76 not found (message Type_93)
21:00:14 Server: Object 3:77 not found (message Type_121)
21:00:14 Server: Object 3:78 not found (message Type_127)
21:00:14 Server: Object 3:79 not found (message Type_114)
21:00:14 Server: Object 3:80 not found (message Type_121)
21:00:14 Server: Object 3:82 not found (message Type_127)
21:00:14 Server: Object 3:83 not found (message Type_121)
21:00:14 Server: Object 3:84 not found (message Type_114)
21:00:14 Server: Object 3:93 not found (message Type_450)
21:00:14 Server: Object 3:85 not found (message Type_93)
21:00:14 Server: Object 3:86 not found (message Type_114)
21:00:14 Server: Object 3:87 not found (message Type_121)
21:00:14 Server: Object 3:88 not found (message Type_121)
21:00:14 Server: Object 3:89 not found (message Type_127)
21:00:14 Server: Object 3:90 not found (message Type_121)
21:00:14 Server: Object 3:91 not found (message Type_114)
21:00:14 Server: Object 3:92 not found (message Type_121)
21:00:14 Server: Object 3:94 not found (message Type_127)
21:00:14 Server: Object 3:95 not found (message Type_121)
21:00:14 Server: Object 3:96 not found (message Type_114)
21:00:14 Server: Object 3:97 not found (message Type_121)
21:00:14 Server: Object 3:98 not found (message Type_121)
21:00:14 Server: Object 3:99 not found (message Type_127)
21:00:14 Server: Object 3:100 not found (message Type_114)
21:00:14 Server: Object 3:106 not found (message Type_93)
21:00:14 Server: Object 3:101 not found (message Type_121)
21:00:14 Server: Object 3:102 not found (message Type_121)
21:00:14 Server: Object 3:103 not found (message Type_127)
21:00:14 Server: Object 3:104 not found (message Type_121)
21:00:14 Server: Object 3:105 not found (message Type_114)
21:00:14 Server: Object 3:108 not found (message Type_451)
21:00:14 Server: Object 3:107 not found (message Type_121)
21:00:14 Server: Object 3:110 not found (message Type_114)
21:00:14 Server: Object 3:109 not found (message Type_121)
21:00:14 Server: Object 3:115 not found (message Type_127)
21:00:14 Server: Object 3:114 not found (message Type_121)
21:00:14 Server: Object 3:122 not found (message Type_115)
21:00:14 Server: Object 3:120 not found (message Type_450)
21:00:14 Server: Object 3:121 not found (message Type_452)
21:00:14 Server: Object 3:111 not found (message Type_130)
21:00:14 Server: Object 3:112 not found (message Type_132)
21:00:14 Server: Object 3:113 not found (message Type_121)
21:00:14 Server: Object 3:119 not found (message Type_121)
21:00:14 Server: Object 3:116 not found (message Type_114)
21:00:14 Server: Object 3:117 not found (message Type_130)
21:00:14 Server: Object 3:118 not found (message Type_132)
21:00:14 Server: Object 3:126 not found (message Type_450)
21:00:14 Server: Object 3:127 not found (message Type_128)
21:00:14 Server: Object 3:125 not found (message Type_450)
21:00:14 Server: Object 3:128 not found (message Type_115)
21:00:14 Server: Object 3:132 not found (message Type_114)
21:00:14 Server: Object 3:129 not found (message Type_352)
21:00:14 Server: Object 3:142 not found (message Type_115)
21:00:14 Server: Object 3:141 not found (message Type_450)
21:00:14 Server: Object 3:140 not found (message Type_452)
21:00:14 Server: Object 3:139 not found (message Type_450)
21:00:14 Server: Object 3:138 not found (message Type_450)
21:00:14 Server: Object 3:137 not found (message Type_115)
21:00:14 Server: Object 3:136 not found (message Type_450)
21:00:14 Server: Object 3:135 not found (message Type_451)
21:00:14 Server: Object 3:130 not found (message Type_114)
21:00:14 Server: Object 3:131 not found (message Type_352)
21:00:14 Server: Object 3:133 not found (message Type_121)
21:00:14 Server: Object 3:134 not found (message Type_121)
21:00:14 Server: Object 3:145 not found (message Type_450)
21:00:14 Server: Object 3:150 not found (message Type_121)
21:00:14 Server: Object 3:153 not found (message Type_115)
21:00:14 Server: Object 3:149 not found (message Type_450)
21:00:14 Server: Object 3:152 not found (message Type_450)
21:00:14 Server: Object 3:151 not found (message Type_452)
21:00:14 Server: Object 3:146 not found (message Type_450)
21:00:14 Server: Object 3:147 not found (message Type_433)
21:00:14 Server: Object 3:148 not found (message Type_115)
21:00:14 Server: Object 3:32 not found (message Type_105)
21:00:14 Server: Object 3:155 not found (message Type_259)
21:00:14 Server: Object 3:160 not found (message Type_455)
21:00:14 Server: Object 3:165 not found (message Type_455)
21:00:14 Server: Object 3:164 not found (message Type_455)
21:00:14 Server: Object 3:163 not found (message Type_455)
21:00:14 Server: Object 3:162 not found (message Type_455)
21:00:14 Server: Object 3:161 not found (message Type_455)
21:00:14 Server: Object 3:157 not found (message Type_93)
21:00:14 Server: Object 3:154 not found (message Type_131)
21:00:14 Server: Object 3:159 not found (message Type_115)
21:00:14 Server: Object 3:158 not found (message Type_452)
21:00:14 Server: Object 3:156 not found (message Type_121)
21:00:14 Server: Object 3:47 not found (message Type_105)
21:00:14 Server: Object 3:62 not found (message Type_105)
21:00:14 Server: Object 3:167 not found (message Type_115)
21:00:14 Server: Object 3:166 not found (message Type_352)
21:00:14 Server: Object 3:171 not found (message Type_93)
21:00:14 Server: Object 3:168 not found (message Type_450)
21:00:14 Server: Object 3:170 not found (message Type_127)
21:00:14 Server: Object 3:172 not found (message Type_115)
21:00:14 Server: Object 3:169 not found (message Type_450)
21:00:14 Server: Object 3:175 not found (message Type_141)
21:00:14 Server: Object 3:177 not found (message Type_455)
21:00:14 Server: Object 3:176 not found (message Type_141)
21:00:14 Server: Object 3:47 not found (message Type_105)
21:00:14 Dimensions in class B_static_AA_F should be an array of size 2.
21:00:14 Dimensions in class B_GMG_01_high_F should be an array of size 2.
21:00:15 Dimensions in class B_static_AT_F should be an array of size 2.
21:00:15 Dimensions in class B_HMG_01_high_F should be an array of size 2.
/home/arma3server/lgsm/functions/command_debug.sh: line 102: 19202 Segmentation fault      (core dumped) ${executable} ${parms//\\;/;}

**Comment by Stigmate-Fr** [comment]
Same here.
No problem for launch and play on home dedicated server. (with [TADST](http://www.armaholic.com/page.php?id=11655) on a windows pc)
But on my rented linux server, crash with the same message. Every other missions play well.
Serv rented for this missions... Hell !

**Comment by amanuense** [comment]
if you are interested I have a fork for dedicated server tested on linux and windows dedicated servers. there are some issues on linux that I have reported to BI and might or might not be fixed.

**Comment by CurePindal** [comment]
>if you are interested I have a fork for dedicated server tested on linux and windows dedicated servers.

And why dont you supply those fixes via Pull Request?

**Comment by amanuense** [comment]
TL;DR: too many changes too little time

anyone could create a pull request from my fork into the main fork. I'll be happy to provide guidance to any questions you might have.

originally I wanted to do that once I added HC support but currently I don't have the time to try to merge with the changes to the main project (turns out newborns and new jobs on a different city remove a lot of your free time). the projects deviated more than a year ago and I have several changes to the "framework" since I'm slowly preparing the code to be able to use Headless Client for Opfor, improving the logic, optimizing stuff, securing the code etc.
I could try to do a pull request but with the little free time I have I won't be able to merge the changes any time soon. I have been idle for months and only one person is helping me (Caius Ajiz https://github.com/CaiusAjiz )

**Comment by CurePindal** [comment]
Tried to reproduce crashs on a dedicated server without success. In my tests the server and mission startup shows no issues.
(v0.6 release version, dedicated Win10 server)

**Comment by amanuense** [comment]
Cai and I can reproduce in Linux server

**Comment by CurePindal** [comment]
Ok, and because you say you fixed it (without offering a pull request) I assume you know whats causing it.. So whats causing the crashs?

**Comment by amanuense** [comment]
when refortifying the FOBs the spawn position sometimes was a little
underground and caused havoc with the physics engine (reported the issue to
BI I don't think they fixed it completely).

we have seen the issue a couple of times but only on the linux dedicated
server.

Oscar

**Comment by amanuense** [comment]
forgot to mention that we checked only using our fork which has several changes to improve dedicated server support

**Comment by CaiusAjiz** [comment]
Looks like this has been fixed by @ultranoobian at some point. I can successfully run this on a dedi now.

**Comment by amanuense** [comment]
Holy s#!t this thing still alive
